data/clients.yaml

```diff
@@ -178,7 +187,9 @@ clients:
     features:
       report_jaeger_thrift_udp: yes
       report_jaeger_thrift_http: yes
-      report_zipkin_thrift_http: no
+      report_zipkin_thrift_http: yes # https://github.com/jaegertracing/jaeger-client-java/tree/v0.35.5/jaeger-zipkin#thrift
+      report_thrift_compact: yes
+      report_thrift_binary: yes
```

Comment:
I think this is confusing in this form, because it's not clear how binary/compact protocol is related to udp/http transport. I believe HTTP always uses `binary` on the backend, and Java client is hardcoded to use `compact` over UDP: https://github.com/jaegertracing/jaeger-client-java/blob/74adce4c4b4c507ccfb010ffb06c95973cc97c6a/jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/UdpSender.java#L47

Reply:
Maybe the matrix should be explicit (even though it gets somewhat verbose):
```
      report_jaeger_thrift_compact_udp: yes
      report_jaeger_thrift_binary_udp: no ### yes for Node only
      report_jaeger_thrift_compact_http: #### always no for all clients
      report_jaeger_thrift_binary_http: yes
      report_zipkin_thrift_binary_http:
```

Reply:
> I believe HTTP always uses binary on the backend

This seems indeed to be the case:
https://github.com/jaegertracing/jaeger/blob/29e71310f3f275def3e07d257901b6e89ad0818e/cmd/collector/app/http_handler.go#L82

https://github.com/apache/thrift/blob/ded326101af3c6c9daad9814ce6404d385f36a1f/lib/go/thrift/deserializer.go#L31-L39

Reply:
Does it still make sense to have network protocol + thrift format, given that all HTTP clients are sending Thrift Binary?

Reply:
yes, because UDP uses both binary and compact, so I would rather be consistent and have report_jaeger_thrift_binary_http, even though we won't have any report_jaeger_thrift_compact_http